detailed_docs/API/OnDriverDistraction/OnDriverDistraction_TRS.md

```diff
@@ -153,16 +154,49 @@ NOT transfer OnDriverDistraction(DD=OFF) after updating `lockScreenDismissalEnab
         </param>
     </function>
     <!-- newly added parameter -->
-    <param name="lockScreenDismissalEnabled" type="Boolean" mandatory="false">
+  <param name="lockScreenDismissalEnabled" type="Boolean" mandatory="false">
     <description>
       If enabled, the lock screen will be able to be dismissed while connected to SDL, allowing users 
-      the ability to interact with the app. Dismissals should include a warning to the user and ensure 
-      that they are not the driver.
+      the ability to interact with the app.
+    </description>
+  </param>
+  <param name="lockScreenDismissalWarning" type="String" mandatory="false">
+    <description>
+      Warning message to be displayed on the lock screen when dismissal is enabled.
+      This warning should be used to ensure that the user is not the driver of the vehicle, 
+      ex. `Swipe up to dismiss, acknowledging that you are not the driver.`.
+      This parameter must be present if "lockScreenDismissalEnabled" is set to true.
     </description>
   </param>
- ```  
- 
-2. HMI_API
+</function>
+```  
+
+1.1
+In case `lockScreenDismissalEnabled` is set to `true`  
+a swipe up gesture to dismiss the lock screen must be allowed 
+
+1.2  
+The `lockScreenDismissalWarning` text must be pulled from the `textBody` field of the appropriate consumer-friendly message (corresponding to the active UI language), all other fields will be ignored.
+
+1.3  
+The language of `lockScreenDismissalWarning` text should correspond to the active UI language (RegisterAppInterface`hmiDisplayLanguageDesired`).
+
+1.4  
+In case active UI language is different from "EN_US" and `lock_screen_dismissal_warning`(value of PT) doesn't contain this language,  
+the text of the `lockScreenDismissalWarning` will be diplayed in default "EN_US".  
+
+_Note: it is OEM responsibility to provide message translations._
+
+1.5
+
+In case `lockScreenDismissalEnabled` is set to `true`  
+and `lockScreenDismissalWarning` is absent
```

Comment:
@KhrystynaDubovyk `LockScreenDismissalWarning` message is absent at all in Policy DB

Reply:
@dboltovskyi, edited in https://github.com/smartdevicelink/sdl_requirements/pull/126/commits/3a9a8062bc46324deb81ffc1a61540f1fe998bd5